Tests/ML/pipelines/test_inference_smallimages.py

```diff
@@ -59,15 +59,6 @@ def run_inference_on_unet(size: TupleInt3) -> None:
         image_util.check_array_range(p)
 
 
-def test_inference_on_too_small_image() -> None:
-    """
```

Comment:
Why delete a test?

Reply:
Because images will never be too small after merging
- #681

Reply:
The other PR does not have tests. Is it not possible to have a test to demonstrate that this is correct?

Reply:
I believe all this is covered by the tests in https://github.com/microsoft/InnerEye-DeepLearning/blob/main/Tests/ML/models/architectures/test_crop_size_multiple.py